### ISSUE TITLE
how to prevent my sword to be used for block breaking?

### ISSUE THREAD

**Issue by MSandro** [Lizard-Of-Oz/Inventorio#121]
If I have a Sword in the Tool Belt it will used to break e.g. Leaves.
Tested in 1.18.2 Fabric.
Is there a config option to prevent this?

**Comment by Lizard-Of-Oz** [comment]
Nope. You can manually hold an incompatible tool in your hand to prevent that from happening.
